packages/@aws-cdk-testing/cli-integ/lib/with-cli-lib.ts

```diff
@@ -134,6 +134,7 @@ export class CliLibIntegrationTestFixture extends TestFixture {
 __EOS__`], {
       ...options,
       modEnv: {
+        ...this.cdkShellEnv(),
```

Comment:
Unrelated but required for when tests start running with atmosphere. It passes the allocated environment credentials down to the CDK subprocess.

Reply:
See https://github.com/aws/aws-cdk-cli-testing/blob/84ec710b8caf2de54d3a91d6324837edb0fc3af4/packages/%40aws-cdk-testing/cli-integ/lib/with-cdk-app.ts#L516